src/components/Item.js

```diff
@@ -1,11 +1,11 @@
 import React from "react";
-import ListToggle from "./ListToggle";
+import ListToggleContainer from "../containers/ListToggleContainer";
 
 function Item(props) {
-  debugger;
+  // debugger;
   let name = "";
   let backDrop = "http://image.tmdb.org/t/p/original" + props.movie.backdrop_path;
```

Comment:
use template strings instead of ES5-style string concatenation for building URLs.